packages/scratch-vm/src/extensions/scratch3_musiccreation/index.js

```diff
@@ -12,6 +12,7 @@ const AnalysisHelpers = require('./analysishelpers');
 const MusicPlayers = require('./musicplayer')
 const textRender = require('./textrender');
 const regeneratorRuntime = require("regenerator-runtime");
+const { u } = require('./letters');
```

Comment:
Is this actually required? Seems maybe like one of those funny things where the IDE tried to include something for you